packages/aws-cdk-lib/aws-stepfunctions/test/activity.test.ts

```diff
@@ -70,4 +71,72 @@ describe('Activity', () => {
     });
 
   });
+
+  test('instantiate Activity with EncryptionConfiguration using Customer Managed Key', () => {
+    // GIVEN
+    const stack = new cdk.Stack();
+    const kmsKey = new kms.Key(stack, 'Key');
+
+    // WHEN
+    new stepfunctions.Activity(stack, 'Activity', {
+      kmsKey: kmsKey,
+      kmsDataKeyReusePeriodSeconds: cdk.Duration.seconds(75),
+    });
+
+    // THEN
+    Template.fromStack(stack).hasResourceProperties('AWS::StepFunctions::Activity', {
+      Name: 'Activity',
+      EncryptionConfiguration: Match.objectLike({
+        KmsKeyId: { 'Fn::GetAtt': ['Key961B73FD', 'Arn'] },
+        KmsDataKeyReusePeriodSeconds: 75,
+        Type: 'CUSTOMER_MANAGED_KMS_KEY',
+      }),
+    });
+  });
+
+  test('instantiate Activity with invalid KmsDataKeyReusePeriodSeconds throws error', () => {
```

Comment:
Start test name with capital letter?

Reply:
Will update all tests to include upper case.